readme_template.md

```diff
@@ -43,9 +43,13 @@ To run unit tests, in the top level directory, just run:
 
     python testUpdateHostsFile.py
 
-**Note** if you are using Python 2, you must first install the `mock` library:
+**Note** if you are using Python 2, please install the dependencies with:
 
```

Comment:
We haven't been tagging releases here.  I'm thinking of the support load coming from those who do a `git pull` and suddenly crash.  So I thing we'll need a major release tag once this is merged.

Reply:
@funilrys sorry this was me truncating.  In everyday work, I use [git-friendly](https://github.com/jamiew/git-friendly), which reduces most of my command line interactions to simply `pull`, `push`, `branch`, and `merge` terminal one-liners.  Behind the scenes, git-friendly performs `git stash` and `git rebase` and many other oh-so-sensible things to make the whole experience completely seamless.

I think [git-friendly](https://github.com/jamiew/git-friendly) is awesome, and I don't understand why it doesn't have 20,000 stars on Github.  cc @jamiew